imgui-knobs.cpp

```diff
@@ -9,6 +9,46 @@
 
 namespace ImGuiKnobs {
     namespace detail {
+
+
+
+    //Re-maps a number from one range to another.
+    //Numbers outside the range are not clamped to 0 and 1.
+    template<typename T>
+    inline T Map(const T& aValue,const T& aIStart,const T& aIStop,const T& aOStart,const T& aOStop)
+    {
+      return aOStart + (aOStop - aOStart) * ((aValue - aIStart) / (aIStop - aIStart));
+    }
+
+
+    // Re-Maps clamped
+    template<typename T>
+    inline T MapC(const T& aValue,const T& aIStart,const T& aIStop,const T& aOStart,const T& aOStop)
```

Comment:
Please use kebab case, and maybe use a more descriptive name, like `map_range_clamped`. Also don't add the `a` for paramteres, just name them `value`, `in_start`, `out_stop`, etc.

Reply:
Sure, I'll try to do this week end.